include/meta/meta.hpp

```diff
@@ -3162,12 +3173,15 @@ namespace meta
         /// \f$ O(log(N)) \f$.
         /// \ingroup integral
         template <typename T, T N>
-        using make_integer_sequence =
+        using make_integer_sequence = _t<
             _t<detail::coerce_indices_<T, 0, make_index_sequence<static_cast<std::size_t>(N)>>>;
```

Comment:
@ldionne I suppose @gnzlbg means [this post](https://www.reddit.com/r/cpp/comments/4k57tk/boost_mpl_dead/) where I mention metabench in one of my replies.

To answer the question, metabench results are updated daily, however there are no benchmarks as of yet that specifically compare the performance of generating sequences.